cpp/Integri/XsdComparator/Reporter.cpp

```diff
@@ -0,0 +1,33 @@
+#include "Reporter.h"
+#include <fstream>
+
+void Reporter::Add(ReporterRow* row)
+{
+    report.push_back(row);
+}
+
+void Reporter::Add(ReporterRowStatus reportRowStatus, int depth, std::string nodeName, std::string nodeValueOfName)
+{
+    if (IsAcceptableNode(nodeName))
+    {
+        Add(new ReporterRow(reportRowStatus, depth, nodeName, nodeValueOfName));
```

Comment:
This `new `has to have a delete somewhere! Or else, consider using smart pointers as suggested somewhere else in my review.